test/fixtures/user/search.js

```diff
@@ -0,0 +1,8 @@
+/**
+ * Sample search values for tests
+ * @return  {object}
+ */
+
+module.exports = () => {
+  return ["an", "AN", "", "mu"];
+};
```

Comment:
Is this adding value?

Reply:
changed into an object, i believe keeping all of the data used for testing at single place would be good